src/services/sig/index.ts

```diff
@@ -206,8 +206,18 @@ export class SigService implements ISigService {
   public async listSigs(
```

Comment:
Maybe the comment of the function should be modified.

Reply:
Good catch! Fixed.

Reply:
Thanks for you review.